courses/free-courses-bn.md

```diff
@@ -74,6 +74,7 @@
 ### Go
 
 * [Golang কোডিং বুট ক্যাম্প ক্লাস](https://youtube.com/playlist?list=PLZij6bgEHkTXRakAtponkmP2CmlTTKlxl) - MASTER-ACADEMY
+* [Go Bangla Tutorials 2022](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rtasmmoS_EWXdg0X-eX_x8) - Anisul Islam
```

Comment:
This playlist is not yet finished, should we add `(:construction: *in process*)` even though it's not a book?

Reply:
It's very close to the end. But you could do what would be the best to do.